p2panda-rs/src/lib.rs

```diff
@@ -80,3 +80,12 @@ pub trait Validate {
     /// Validates p2panda data type instance.
     fn validate(&self) -> Result<(), Self::Error>;
 }
+
+/// Init pretty_env_logger before the test suite runs to handle logging outputs.
+/// Uses the `ctor` crate to define a global constructor function.
```

Comment:
It might be good to document this a bit more. How can consumers of the logs receive them? Maybe this could go into the handbook? Or readme?

Reply:
Maybe also make explicit that "global contructor function" means? Not sure, maybe only I don't know what that is :D